### DISCUSSION TITLE
Solidity資源分享

### DISCUSSION THREAD

**Discussion by manipulateed** [1101-GS4719-C/Python-Community-GS4719-C#733] [General]
Solidity是一種合約導向式語言，被應用於各種不同的區塊鏈平台，以下的影片將帶大家進入這個領域，
我覺得這樣有系統的課程對於學習非常有幫助，希望大家能有所收穫。

https://www.youtube.com/watch?v=z2FEikbDqoA&t=6s

**Comment by williamyen0623** [comment]
想學習撰寫智能合約的基礎可以從Solidity開始

**Comment by danwan1027** [comment]
solidity 的學習資源相對其他語言較少，感謝大大分享！

**Comment by Chiway1229** [comment]
感謝有你的介紹 最近正好對DeFi跟Solidity有興趣
看完這些影片後相信能對我的能力有所提升

**Comment by evanchou1204** [comment]
最近剛加入區塊鏈研究社，努力往不成為韭菜的路上前進，感謝你分享的教學🥹

**Comment by Pietra1226** [comment]
常常對於導向式語言的概念很模糊，這也是我第一次聽到這種語言，未來有機會會努力學習~

**Comment by Yunhans** [comment]
第一次聽到合約導向式語言，之後來研究看看🧐
